eessi/testsuite/tests/apps/numpy/src/np_ops.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from time import time
+
+
+print('NumPy version:', np.__version__)
+
+# Let's take the randomness out of random numbers (for reproducibility)
+np.random.seed(534)
+
+size = 8192
+A, B = np.random.random((size, size)), np.random.random((size, size))
+E = np.random.random((int(size / 2), int(size / 4)))
+F = np.random.random((int(size / 2), int(size / 2)))
+F = np.dot(F, F.T)
+G = np.random.random((int(size / 2), int(size / 2)))
+J = (F + F.T) / 2.
+
+# Matrix multiplication
+N = 20
```

Comment:
It would be nice to make `size` and `N` in the arguments of the script, so that they can be set from the ReFrame test. You can then set `20` and `8192` as the defaults, but it would allow overriding via `--setvar` for example. If you do that, please do add a remark to the `required_mem_per_node` that this memory requirement is _based_ on a size `8192` and may need to be adapted for other sizes.